Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/UI/RecordingServiceVisual.cs

```diff
@@ -237,13 +238,15 @@ public void Show()
                 StopRecording();
             }
 
-            gameObject.SetActive(true);
+            _recordButton.gameObject.SetActive(true);
+            _previewButton.gameObject.SetActive(true);
```

Comment:
>_previewButton.gameObject.SetActive(true); [](start = 12, length = 42)

Seems strange we always enable previewButton here, even if preview is not available.

Perhaps we can toggle on/off the parent of both these buttons, and let the various states (Recording, preview enabled, etc) toggle whether these buttons are enabled/disabled